eng/scripts/Update-VcpkgPort.ps1

```diff
@@ -89,30 +94,34 @@ if ($LASTEXITCODE -ne 0) {
 Write-Host "git reset HEAD^"
```

Comment:
OK it looks like you are already doing the git reset in this script but then you are doing another commit below with the change log information.